kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaServiceReconciler.java

```diff
@@ -73,15 +75,26 @@ public List<EventSource<?, KafkaService>> prepareEventSources(EventSourceContext
                 .withPrimaryToSecondaryMapper(kafkaServiceToSecret())
                 .withSecondaryToPrimaryMapper(secretToKafkaService(context))
                 .build();
+
         InformerEventSourceConfiguration<ConfigMap> serviceToConfigMap = InformerEventSourceConfiguration.from(
                 ConfigMap.class,
                 KafkaService.class)
                 .withName(CONFIG_MAPS_EVENT_SOURCE_NAME)
                 .withPrimaryToSecondaryMapper(kafkaServiceToConfigMap())
                 .withSecondaryToPrimaryMapper(configMapToKafkaService(context))
                 .build();
+
+        InformerEventSourceConfiguration<Kafka> serviceToStrimziKafka = InformerEventSourceConfiguration.from(
+                Kafka.class,
+                KafkaService.class)
+                .withName(CONFIG_MAPS_EVENT_SOURCE_NAME)
```

Comment:
The event source name seems wrong. I would have expected something like STRIMZI_KAFKA_EVENT_SOURCE_NAME